backends/ceed-backend-weak.c

```diff
@@ -19,11 +19,12 @@ static int CeedInit_Weak(const char *resource, Ceed ceed) {
 // This function provides a debug target for weak symbols
 static int CeedRegister_Weak(const char *name, int num_prefixes, ...) {
   va_list prefixes;
+  int     ierr;
+
   va_start(prefixes, num_prefixes);
-  int ierr;
   for (int i = 0; i < num_prefixes; i++) {
     const char *prefix = va_arg(prefixes, const char *);
-    CeedDebugEnv("** Weak Register: %s", prefix);
+    CeedDebugEnv("Weak Register: %s", prefix);
```

Comment:
This is here so when you run with `CEED_DEBUG=1` environment variable the list of registered backends is aligned and the weakly registered backends (the ones not actually built) stand out.

```suggestion
    CeedDebugEnv("** Weak Register: %s", prefix);
```

Could also do this I suppose
```suggestion
    CeedDebugEnv("Weak Register   : %s", prefix);
```

Or something else to add 3 characters before the name of the backend

Reply:
Fixed as well in [695c241](https://github.com/CEED/libCEED/pull/1279/commits/695c241f98c558443702e1226c36e4fb82ee64da).